datamodel/core.py

```diff
@@ -207,9 +225,21 @@ def get_schema(cls, **kwargs):
         """
         return cls.__get_schema_class__(**kwargs)
 
+    @classmethod
+    def validate(cls, data, context=None, many=None, partial=None, unknown=None):
```

Comment:
Can you add a dedicated tests to chack that the unknow param is recursively applied to nested schemas on validate and cleaned up after